cinema_service/settings.py

```diff
@@ -137,3 +134,9 @@
 # https://docs.djangoproject.com/en/4.0/ref/settings/#default-auto-field
 
 DEFAULT_AUTO_FIELD = "django.db.models.BigAutoField"
+
+REST_FRAMEWORK = {
+    "DEFAULT_AUTHENTICATION_CLASSES": [
+        "rest_framework_simplejwt.authentication.JWTAuthentication",
```

Comment:
The `DEFAULT_AUTHENTICATION_CLASSES` is set to use JWT authentication. Ensure that your JWT settings, such as token expiration and secret key, are properly configured in your Django settings.